core/templates/dev/head/pages/base.html

```diff
@@ -95,6 +95,7 @@
         SITE_NAME: JSON.parse('{{SITE_NAME|js_string}}'),
         SYSTEM_USERNAMES: JSON.parse('{{SYSTEM_USERNAMES|js_string}}'),
         TEMPLATE_DIR_PREFIX: JSON.parse('{{TEMPLATE_DIR_PREFIX|js_string}}'),
+        SITE_FEEDBACK_FORM_URL: JSON.parse('{{SITE_FEEDBACK_FORM_URL|js_string}}'),
```

Comment:
Move this to line 95 (following alphabetic order!). (Sorry I didn't mention this in the last review)

Reply:
Done!